plugins/scanners/fossid/src/main/kotlin/FossId.kt

```diff
@@ -907,7 +907,7 @@ class FossId internal constructor(
                 source = descriptor.id,
```

Comment:
Commit message:
Please clarify why this is required.

Something like:
```
While a pending identification is not per se an issue, because it is a call
to action and not a finding, some users might want that ORT shows it with
the severity ERROR to ensure pending identifications are not overlooked in
CI/CD pipelines or compliance workflows where only errors are considered
blocking issues.
```